### PR TITLE
Fix/nfersup 25/pause on unrecoverable error

### PR DESCRIPTION
**Pull request by hutnikau**
Related to : https://github.com/oat-sa/extension-tao-testqti/pull/1606
Please see the descriptions in #1606

### CODE REVIEW

File: src/proxy/cache/proxy.js [hunk 1]
```diff
@@ -207,6 +211,12 @@ export default _.defaults(
                                                         ? actionResult.requestParameters.actionId
                                                         : null;
 
+                                                if (!actionResult.success && self.actionRejectPromises[actionId]) {
+                                                    var error = new Error(actionResult.message);
```
Comment (krampstudio) [inline src/proxy/cache/proxy.js]: `const error`

File: src/proxy/cache/proxy.js [hunk 2]
```diff
@@ -207,6 +211,12 @@ export default _.defaults(
...
+                                                    error.unrecoverable = true;
+                                                    reject(error);
```
Comment (krampstudio) [inline src/proxy/cache/proxy.js]: please return before `reject` to prevent unexpected code execution

**Comment by zagovorichev** [comment]
Not sure that I did everything properly, but:
1. Started new delivery execution

2. open second tab with the same de
3. on the 1 tab I answered yes no yes and press next:

4. on the second tab answered no yes no for the same first item:

nothing was changed
5. 2 tab: go back and do the same no yes no:

**Comment by zagovorichev** [comment]
@hutnikau check my comments, please

**Pull request merged**